.github/workflows/main.yml

```diff
@@ -16,8 +16,9 @@ on:
 
 env:
   IMAGE: frontstage
-  REGISTRY_HOSTNAME: eu.gcr.io
+  REGISTRY_HOSTNAME: europe-west2-docker.pkg.dev
   HOST: ${{ secrets.GOOGLE_PROJECT_ID }}
+  GAR_REPOSITORY: ${{ secrets.GAR_REPOSITORY }}
   RELEASE_HOST: ${{ secrets.RELEASE_PROJECT_ID }}
```

Comment:
I'm finding these confusing and think we should take the opportunity to rationalise and rename.

The `HOST` and `RELEASE_HOST` are incorrectly named and misleading. We should rename `RELEASE_HOST` to `GAR_GOOGLE_PROJECT_ID` and rename `HOST` to `SPINNAKER_GOOGLE_PROJECT_ID`.

Reply:
Yeah thats true, I'll update them for this one and add it to the next card

Reply:
Looks much better, the vars now accurately reflect their purpose.